model/minsky.cc

```diff
@@ -1078,17 +1078,27 @@ namespace minsky
         ds[j]=1;
         for (size_t i=0; i<equations.size(); ++i)
           equations[i]->deriv(&df[0], df.size(), &ds[0], sv, &flow[0]);
-        vector<double> d(stockVars.size());
-        evalGodley.eval(&d[0], &df[0]);
+        double d[stockVars.size()];
```

Comment:
This is not standard C++.

Reply:
Whoops. i meant to change it back and forgot...